### PR TITLE
Fix RELEASES overwrite issue

### PR DESCRIPTION
**Pull request by hoffmabc**
- Prevents the RELEASES file being overwritten by the client only binary generation. 
- Adds back electron-winstaller specific older version because otherwise 7zip cannot build binaries

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -32,7 +32,6 @@ deploy:
     - "dist/win32/RELEASES"
     - "dist/win64/*.exe"
     - "dist/win64/*.nupkg"
-    - "dist/win64/RELEASES-x64"
```
Comment (hoffmabc) [inline .travis.yml]: We remove this because squirrel does not care about this file.

File: build.sh [hunk 2]
```diff
@@ -222,6 +224,8 @@ case "$TRAVIS_OS_NAME" in
     signcode -t http://timestamp.digicert.com -a sha1 -spc .travis/ob1.cert.spc -pvk .travis/ob1.pvk -n "OpenBazaar $PACKAGE_VERSION" dist/win64/OpenBazaar2-$PACKAGE_VERSION-Setup-64.exe
     signcode -t http://timestamp.digicert.com -a sha1 -spc .travis/ob1.cert.spc -pvk .travis/ob1.pvk -n "OpenBazaarClient $PACKAGE_VERSION" dist/win64/OpenBazaar2Client-$PACKAGE_VERSION-Setup-64.exe
 
+    mv dist/RELEASES dist/win32/RELEASES
```
Comment (jjeffryes) [inline build.sh]: This is in the win64 section, should it be win64 instead of win32?

Comment (hoffmabc) [reply build.sh]: No we need to wait until all the builds are done and then move the file.

Comment (hoffmabc) [reply build.sh]: I tested the build process locally and it looked good when simulating the Travis build.

**Comment by jjeffryes** [comment]
> @hoffmabc can you update this from master to fix the lint issues?

You're travelling so I did it.

**Comment by hoffmabc** [comment]
Thank you

**Comment by hoffmabc** [comment]
Ok I can’t review my own PR. Are you good to merge?

**Comment by jjeffryes** [comment]
> Ok I can’t review my own PR. Are you good to merge?

Yes, I was just waiting for all the Travis checks to finish.

**Pull request merged**